app/config/question-bank.js

```diff
@@ -1014,13 +1014,104 @@ const questionBank = {
+            },
+          },
+          preValidationKeys: [],
+          fundingPriorities: "",
+          type: "single-answer",
+          minAnswerCount: 1,
+          classes: "govuk-radios--inline govuk-fieldset__legend--l",
+          ineligibleContent: {
+            messageContent:
+              "Your existing store must be signed off as fit for purpose by a structural engineer at full application.",
+            messageLink: {
+              url: "https://www.gov.uk/government/collections/rural-payments-and-grants",
+              title: "See other grants you may be eligible for.",
+            },
+          },
+          sidebar: {
+            values: [
+              {
+                heading: "Eligibility",
+                content: [
+                  {
+                    para: "A store is no longer fit for purpose if it has reached the end of its design life and may be at risk of leaks or failure. To cover an existing store, you must confirm at full application that the store is structurally suitable to cover and meet regulations and build standards.",
+                  },
+                ],
+              },
+            ],
+          },
+          validate: [
+            {
+              type: "NOT_EMPTY",
+              error:
+                "Select if your existing store you want to cover is fit for purpose",
+            },
+          ],
+          answers: [
+            {
+              key: "fit-for-purpose-A1",
+              value: "Yes",
+            },
+            {
+              key: "fit-for-purpose-A2",
+              value: "No",
+            },
+          ],
+          yarKey: "fitForPurpose",
+        },
+        {
+          key: "fit-for-purpose-conditional",
+          order: 145,
+          url: "fit-for-purpose-conditional",
+          backUrl: "fit-for-purpose",
+          nextUrl: "estimated-grant",
+          maybeEligible: true,
+          preValidationKeys: [],
+          maybeEligibleContent: {
+            messageHeader:
+              "You may be able to apply for a grant from this scheme",
+            messageContent:
+              "You will not get grant funding for a cover if the existing store you want to cover is not fit for purpose.",
+            warning: {
+              text: "To apply for a cover, your existing store must be signed off as fit for purpose by a structural engineer at full application.",
+            },
+            extraMessageContent: `<p>You can continue to check your eligibility for grant funding to build a new store, or expand or replace an existing store.</p>`,
+          },
+          yarKey: "FitForPurposeCondition",
+        },
         {
           key: "project-type",
-          order: 110,
+          order: 140,
           title: "How will you increase your storage capacity?",
           baseUrl: "project-type",
           backUrl: "applying-for",
-          nextUrl: "cover",
+          nextUrl: "grand-funded-cover",
```

Comment:
these should all be **grant-funded-cover**. Looks like a slight spelling error to be fixed

Reply:
This would need to be changed in this file (for both url and yarKey etc), as well as in Dora/email, gapi-service and tests

Reply:
Alright, l will fix it.